components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Window.kt

```diff
@@ -40,5 +40,5 @@ fun Window.setNavigationBarTheme(@ColorInt color: Int) {
  * Retrieves a {@link WindowInsetsControllerCompat} for the top-level window decor view.
  */
 fun Window.getWindowInsetsController(): WindowInsetsControllerCompat? {
-    return ViewCompat.getWindowInsetsController(this.decorView)
+    return ViewCompat.getWindowInsetsController(findViewById(Window.ID_ANDROID_CONTENT))
```

Comment:
Could you provide some context why this solves the bug? Couldn't the window still be null?

Reply:
The window is not nullable. From the doc of `ViewCompat.getWindowInsetsController` : 

> @return A {@link WindowInsetsControllerCompat} or {@code null} if the view is neither attached to a window nor a view tree with a decor.

On testing this, sometimes when using `decorview` `getWindowInsetsController` returns null.

Reply:
I will investigate this further, but a better approach to this would be to use a `View` extension and always have as a receiver the view from `setContentView`. That would complicate code when we also need the window to apply changes to the status bar or navigation bar.

Reply:
On further investigation, it seems that the `WindowInsetsControllerCompat` constructor would be the recommended approach.